src/libs/PolicyUtils.ts

```diff
@@ -501,7 +501,7 @@ function getActiveAdminWorkspaces(policies: OnyxCollection<Policy> | null): Poli
 
 /** Whether the user can send invoice */
 function canSendInvoice(policies: OnyxCollection<Policy> | null): boolean {
-    return getActiveAdminWorkspaces(policies).length > 0;
+    return getActiveAdminWorkspaces(policies).some((policy) => policy?.areInvoicesEnabled);
```

Comment:
The check is correct, but maybe we should update it later and leave a TODO for now?
Cause invoicing settings screen isn't ready yet, and this update will block users from invoice creation.
It relates to other places where we check for `areInvoicesEnabled` as well.
Maybe we should confirm it

Reply:
I got you. Let me clarify.

Reply:
Yeah that makes sense, let's put it as a todo and update it when the invoices screen is ready

Reply:
I've commented out some changes to prevent the invoice creation from being blocked.

Commit - 4acdd47.

<details><summary>Video</summary>

https://github.com/user-attachments/assets/8247b2fe-efae-4b6e-8f80-0bb3b508d107

</details> 

cc @madmax330 @VickyStash

Reply:
Can I open the PR for review?

Reply:
@rezkiy37 let's add a link to the blocking issue in TODOs https://github.com/Expensify/App/issues/45175, then it's fine

Reply:
Done - c79964d.